voluptuous/tests/tests.py

```diff
@@ -399,6 +399,30 @@ def test_schema_extend_key_swap():
     assert_true((list(extended.schema)[0], Required))
 
 
+def test_remove():
```

Comment:
@benjaminweb There is also a function called `test_remove` on line 91 so one of these tests isn't running.